### ISSUE TITLE
Error "run the learning"

### ISSUE THREAD

**Issue by Changyiyu** [CEA-LIST/N2D2#24]
Hello,
could you help me solve this error of Run the learning

~/N2D2-master/build/bin$ ./n2d2 "mnist24_16c4s2_24c5s2_150_10.ini" -learn 60000 -log 10000
Option -log: number of steps between logs [10000]
Option -learn: number of backprop learning steps [60000]
Loading network configuration file mnist24_16c4s2_24c5s2_150_10.ini
Warning: to use Frame_CUDA models, N2D2 must be compiled with CUDA enabled.
*** Using Frame model instead. ***
Notice: Unused section sp in INI file
Notice: Unused section sp.Transformation in INI file
Notice: Unused section conv1 in INI file
Notice: Unused section conv2 in INI file
Notice: Unused section fc1 in INI file
Notice: Unused section fc1.drop in INI file
Notice: Unused section fc2 in INI file
Notice: Unused section softmax in INI file
Notice: Unused section softmax.Target in INI file
Notice: Unused section common.config in INI file
Time elapsed: 0.000491815 s
Error: Could not open images file: /local/qwdr7777/n2d2_data/mnist/train-images-idx3-ubyte

**Comment by olivierbichler-cea** [comment]
Hello,
Did you check that the file /local/qwdr7777/n2d2_data/mnist/train-images-idx3-ubyte actually exists?
It is possible that when you downloaded it and unzipped it, there are a dots instead of dashes in the file name.

**Comment by olivierbichler-cea** [comment]
This still seems to be a file related issue to me. Maybe is it the file permissions?

**Comment by Changyiyu** [comment]
Thank you. After I add sudo the problem is solved.

**Comment by prachikashikar** [comment]
Hi @Changyiyu , How did you fix this issue? I am stuck on this.

**Comment by Djip007** [comment]
look like not access allowed on the file...

something like:
```
sudo chmod 777 -R /local/qwdr7777/n2d2_data/mnist/
```
may help but not realy clean...

**Comment by prachikashikar** [comment]
Thank you @Djip007  for replying, but this is nor removing the error . :(

**Comment by Changyiyu** [comment]
Hi
Sorry to reply so late. 
I only added "sudo" to access it, but I think this may not be the answer you need.

**Comment by prachikashikar** [comment]
Thank you for reply @Changyiyu  . But yeah this is not working to fix this issue. So you added sudo and your learning went successful??
